### PR TITLE
Fix#315

### PR DESCRIPTION
**Pull request by 1letter**
- add an upgrade step with a fix for the registry settings
- add a test

### CODE REVIEW

**Comment by 1letter** [comment]
@jenkins-plone-org please run jobs

**Comment by 1letter** [comment]
@jenkins-plone-org please run jobs

**Comment by 1letter** [comment]
@jenkins-plone-org please run jobs

File: plone/app/upgrade/tests/test_fix_registry_settings.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            recordname = f"{new_iface}.{fieldname}"
+            self.assertIn(recordname, records)
+
+def test_suite():
+    from unittest import makeSuite
+    from unittest import TestSuite
+
+    suite = TestSuite()
+    suite.addTest(makeSuite(TestFixRegistrySettings))
+    return suite
```
Comment (gforcada) [inline plone/app/upgrade/tests/test_fix_registry_settings.py:70]: I see that both `test_utils.py` and `test_upgrade.py` have these lines, but seems that they are not needed at all. Running the tests with and without these lines on `test_utils.py` and `test_upgrades.py` leads to the same amount of tests being ran.

**Review by gforcada — COMMENTED** [review]
Thanks! 👍🏾

**Comment by 1letter** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**